ibis/pandas/core.py

```diff
@@ -184,26 +208,33 @@ def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
         post_execute_=functools.partial(
             post_execute,
             scope=scope,
+            timecontext=timecontext,
             aggcontext=aggcontext,
             clients=clients,
             **kwargs,
         ),
         **kwargs,
     )[op]
-
     return result
 
 
 @trace
 def execute_until_in_scope(
-    expr, scope, aggcontext=None, clients=None, post_execute_=None, **kwargs
+    expr,
+    scope,
+    timecontext=None,
+    aggcontext=None,
```

Comment:
type what you can

Reply:
I have typed all appearances of timecontext.